src/ecs/resources/input/button.rs

```diff
@@ -0,0 +1,29 @@
+use engine::{VirtualKeyCode, MouseButton};
+
+use super::local_mouse_button::LocalMouseButton;
+use super::local_virtual_key_code::LocalVirtualKeyCode;
+
+/// A Button is any kind of digital input that the engine supports.
+#[derive(Eq, PartialEq, Debug, Copy, Clone, Serialize, Deserialize)]
+pub enum Button {
+    /// Keyboard keys
+    Key(#[serde(with = "LocalVirtualKeyCode")]
```

Comment:
@torkleyy I had that thought, since glutin types are also winit types I figured this way might be easier so we don't have to write conversions, but ultimately I don't really have strong feelings one way or another.